testing/trino-testing/pom.xml

```diff
@@ -16,6 +16,16 @@
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
     </properties>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.glassfish.jersey.core</groupId>
+                <artifactId>jersey-common</artifactId>
+                <version>2.26</version>
```

Comment:
Don't add the dependency here. It's not directly used by this module. You can work around the dependency plugin false positive by building the string via concatenation, or by using `<ignoredUsedUndeclaredDependencies>` for the plugin.

Reply:
Thanks, I didn't know there can be false positives. I rebased on master now, removed these entries and the dependency plugin doesn't complain anymore, weird.